src/librustdoc/html/render/print_item.rs

```diff
@@ -323,10 +323,10 @@ fn item_module(w: &mut Buffer, cx: &Context<'_>, item: &clean::Item, items: &[cl
 
                 write!(
                     w,
-                    "<tr class=\"{stab}{add}import-item\">\
-                         <td><code>{vis}{imp}</code></td>\
-                         <td class=\"docblock-short\">{stab_tags}</td>\
-                     </tr>",
+                    "<item-left class=\"{stab}{add}import-item\">\
```

Comment:
I wonder about this: I can't find any documentation about this tag (or `item-table`). Is it a custom tag? How is the support for such tags in safari?

Reply:
It is a custom tag, I assumed every browser was supporting them for years ...

Reply:
Currently, I don't have access to a Mac but it looks like the restriction does not impact us because we are not inheriting from other elements which (according to my searches) safari does not support:
https://stackoverflow.com/questions/54160445/custom-elements-works-for-safari-but-not-for-firefox-and-chrome

I mean I could rework it into div's with classes

Reply:
Please use div with classes then. Let's try to avoid potential issues.

Reply:
I did the work